### ISSUE TITLE
Adjust Curate Collections facet to use Source Collection only

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-curate#1410]
Similar to the equivalent Lux ticket 684, update the Collections facet in Curate to only show works related to Source Collections by using their source_collection_id value. Deposit Collections should not be shown. See ticket #1406 for closely related work.

![Collection Facet Curate.png](https://images.zenhubusercontent.com/5b6de3fda32a4f476097b0be/6b5a4de1-c06c-40d0-b4ce-9035109031a1)

As an example, our Dashboard > Works page now has a Source Collection filter which provides the desired behavior:

![Screen Shot 2020-09-14 at 8.55.33 AM.png](https://images.zenhubusercontent.com/5b6de3fda32a4f476097b0be/a6a6e9a6-9f4c-4122-95f2-58c1a1b06a1a)

**Comment by eporter23** [comment]
@wulman16 looks great! I've been adding more source collection IDs to various works and I can see the facet count increasing as I do this. The Langmuir facet is ~1000 works off but I investigated it and there are about that many Langmuir works in test that haven't had their source collection IDs assigned. I also made sure to reindex all the Deposit Collection instances we had so that they had the new indexing updated.
